src/Types/BakerEvent.cs

```diff
@@ -51,6 +51,7 @@ internal static IBakerEvent From(BakerEvent bakerEvent) =>
                 ),
             BakerEvent.EventOneofCase.None =>
                 throw new MissingEnumException<BakerEvent.EventOneofCase>(bakerEvent.EventCase),
+            BakerEvent.EventOneofCase.DelegationRemoved => throw new NotImplementedException(),
```

Comment:
Why? A comment to explain why this isn't implemented would be nice.

Reply:
I tried to split up changes related to fixing build warnings and changes due to P7.
It is implemented in https://github.com/Concordium/concordium-net-sdk/pull/102